liferay-lima/liferay-portal
# LPS-169174 Remove usages of MenuItem in sharing

# What is this about?

https://issues.liferay.com/issues/browse/LPS-169174

Replace usages of MenuItem with DropdownItem in sharing.

# How do I test it?

Share a document or blogs entry. Go to "Shared Content" and verify that all dropdown actions in both the blog entry and the document work.

This is an internal change, so no user visible changes should be noticeable other than markup changes.
